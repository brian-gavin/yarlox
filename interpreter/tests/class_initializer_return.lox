class Class1 {
    init() { }
}

class Class2 {
    init() {
        return;
    }
}

print Class1().init();
print Class2().init();
