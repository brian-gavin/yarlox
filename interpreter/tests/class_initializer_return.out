Class1 instance
Class2 instance
